road/config.ini

```diff
@@ -1,32 +1,35 @@
-[DEFAULT]
-DBF_Files = LOADAM_FINAL.dbf, LOADMD_FINAL.dbf, LOADPM_FINAL.dbf ,LOADEV_FINAL.dbf, LOADEA_FINAL.dbf
+[DBF]
+
 DBF_Directory = Q:/Model Development/Simwrapper Validation/notebooks/roads/Data
+DBF_Files = LOADAM_FINAL.dbf, LOADMD_FINAL.dbf, LOADPM_FINAL.dbf ,LOADEV_FINAL.dbf, LOADEA_FINAL.dbf

```

Comment:
maybe these file names should be calculated on the fly, with the "TOD" (time of day) / "time_period" (AM, MD, ...) specified as another variable
(though this could be more appropriate for another future pull request)

Reply:
also suggest using a linter/formatter to catch e.g. the misformatted comma here (between PM and EV)